static/app/views/eventsV2/utils.tsx

```diff
@@ -507,6 +508,7 @@ export function generateFieldOptions({
   }
 
   if (measurementKeys !== undefined && measurementKeys !== null) {
+    measurementKeys.sort();
```

Comment:
could sort these at the source, but this way we can add more and they'll be sorted without any additional dev work